benchmark/launch_benchmark.sh

```diff
@@ -68,6 +68,16 @@ args=(
     # protein ID to taxon (genome) ID(s) dictionary
     prot2tax_dict_fp
 
+    ## sequence similarity search parameters (BLAST / DIAMOND)
+    # maximum E-value cutoff
+    evalue
+    # minimum identity cutoff (%)
+    pident
+    # minimum query coverage cutoff (%)
+    qcovs
+    # maximum number of target sequences to report
+    max_target_seqs
```

Comment:
These are but standard BLAST (DIAMOND) parameters. I added those so that the user can customize their BLAST search.